app/i18n/lang/fr.json

```diff
@@ -2,23 +2,23 @@
   "enterConferenceNameOrUrl": "Entrez un nom pour votre conférence ou une URL Jitsi",
   "go": "Rejoindre",
   "help": "Aide",
-  "termsLink": "Conditions d'utilisation",
+  "termsLink": "Conditions d’utilisation",
   "privacyLink": "Politique de confidentialité",
   "sendFeedbackLink": "Envoyer des commentaires",
-  "aboutLink": "A propos",
+  "aboutLink": "À propos",
   "sourceLink": "Code source",
-  "versionLabel": "Version: {{version}}",
+  "versionLabel": "Version : {{version}}",
```

Comment:
There should be no space before the colon.

Reply:
https://fr.wikipedia.org/wiki/Espace_ins%C3%A9cable#En_France

> Le deux-points et les guillemets en chevrons constituent une exception en ce que le code typographique français recommande d'insérer devant le deux-points, ou entre les guillemets et le texte qu'ils renferment, non une espace fine insécable mais une espace insécable (espace-mot insécable).

It should even be a non-breaking space. I grew up in this typographic culture, and honestly, I find it difficult to read when there is no space before the colon :smile:

Reply:
Thank you for highlighting and fixing this